Week-9/demo-react-app/src/Home.js

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from "react";
+import './Home.css'
+
+function Note(props){
+
+    function handleCheck(id){
+        var col = document.getElementById(id), check = document.getElementById(id+"_done")
+        if(check.checked)
+            col.style.textDecoration = "line-through";
+        else
+            col.style.textDecoration = "initial";
+    }
+
+    return (
+        <div id={props.id} className="note">
+            <input id={`${props.id}_done`} type="checkbox" onChange={() => handleCheck(props.id)}/>
+            <p>{props.noteText}</p>
+            {/* <button onClick={() => removeNote(props.index)}>Delete</button> */}
+        </div>
+    );
+}
+
+function Header() {
+    return (
+        <header>
+            <h1 id="todoAppText">TodoApp</h1>
+        </header>
```

Comment:
Create this as a seperate component and if any styles are there create a seperate file for it